### PR TITLE
URL Child Block for forms

### PR DESCRIPTION
**Pull request by amithire1201**
### Description
URL Child Block added.

### Types of changes

New feature (non-breaking change which adds functionality) 

### How has this been tested?
Add Forms Block and add URL child  block.

### Checklist:
- [x] My code is tested
- [x] My code follows accessibility standards 
- [ ] My code has proper inline documentation 
- [ ] I've included any necessary tests 
- [ ] I've included developer documentation 
- [x] I've added proper labels to this pull request

### CODE REVIEW

File: src/blocks/forms/child-blocks/url/attributes.js [hunk 1]
```diff
@@ -0,0 +1,14 @@
+const attributes = {
+    block_id: {
+		type: "string"
+    },
+    urlName: {
+        type: "string",
+        default: "URL Name",
```
Comment (RajkiranBagal) [inline src/blocks/forms/child-blocks/url/attributes.js:7]: Add translation

File: src/blocks/forms/child-blocks/url/edit.js [hunk 2]
```diff
@@ -0,0 +1,106 @@
...
+const {
+	Component,
+	Fragment
+} = wp.element
+
+const {
+	PanelBody,
+	SelectControl,
+	RangeControl,
+	TabPanel,
```
Comment (RajkiranBagal) [inline src/blocks/forms/child-blocks/url/edit.js:18]: Remove the components which are not used

File: src/blocks/forms/child-blocks/url/save.js [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+			"uagb-forms-url-wrap",
+			`uagb-block-${ block_id }`,
+		) }>
+				<RichText.Content
+			tagName="div"
+			value={ urlName }
+			className='uagb-forms-url-label'
+			
+			/>
+			<input type="url" required={ urlRequired } className="uagb-forms-url-input"/>
```
Comment (RajkiranBagal) [inline src/blocks/forms/child-blocks/url/save.js:30]: Add a name attr to input

Comment (amithire1201) [reply src/blocks/forms/child-blocks/url/save.js]: Done.

File: src/blocks/forms/child-blocks/url/edit.js (hunk 2 above)
Comment (amithire1201) [reply src/blocks/forms/child-blocks/url/edit.js]: Done.

File: src/blocks/forms/child-blocks/url/attributes.js (hunk 1 above)
Comment (amithire1201) [reply src/blocks/forms/child-blocks/url/attributes.js]: Done.

File: classes/class-uagb-config.php [hunk 4]
```diff
@@ -2224,37 +2224,45 @@ public static function get_block_attributes() {
...
+					),
+					'uagb/forms-url'          => array(
+						'slug'        => '',
+						'title'       => __( 'URL', 'ultimate-addons-for-gutenberg' ),
+						'description' => __( 'This block helps to add URL field.', 'ultimate-addons-for-gutenberg' ),
+						'default'     => true,
 						'attributes'  => array(
-						)
+							'block_id' => '',
+							'urlName'  => 'URL Name',
```
Comment (vrundakansara) [inline classes/class-uagb-config.php:2264]: Let's keep the label just `URL`.
Let's also make it translatable.

File: src/blocks/forms/child-blocks/url/attributes.js [hunk 5]
```diff
@@ -0,0 +1,16 @@
...
+
+const attributes = {
+    block_id: {
+		type: "string"
+    },
+    urlName: {
+        type: "string",
+        default: __( "URL Name" ),
+	},
+	urlRequired : {
```
Comment (vrundakansara) [inline src/blocks/forms/child-blocks/url/attributes.js:11]: Keep it `required`

File: src/blocks/forms/child-blocks/url/attributes.js [hunk 6]
```diff
@@ -0,0 +1,16 @@
+const { __ } = wp.i18n
+
+const attributes = {
+    block_id: {
+		type: "string"
+    },
+    urlName: {
```
Comment (vrundakansara) [inline src/blocks/forms/child-blocks/url/attributes.js:7]: Keep it `name`

File: classes/class-uagb-config.php (hunk 4 above)
Comment (amithire1201) [reply classes/class-uagb-config.php]: Done.

File: src/blocks/forms/child-blocks/url/attributes.js (hunk 5 above)
Comment (amithire1201) [reply src/blocks/forms/child-blocks/url/attributes.js]: Done.

File: src/blocks/forms/child-blocks/url/attributes.js (hunk 6 above)
Comment (amithire1201) [reply src/blocks/forms/child-blocks/url/attributes.js]: Done.

**Review by vrundakansara — APPROVED** [review]

**Pull request merged**